### PR TITLE
Automate minio webhook creation

### PR DESCRIPTION
**Pull request by aliok**
Automate minio webhook creation:
- Use `mc` command in Docker
- Create temp dir to hold `mc` client config, mount it to the `mc` image, run commands, delete the temp dir

### CODE REVIEW

**Comment by aliok** [comment]
cc @pierDipi

File: infra/openshift-manifests/lib.sh [hunk 1]
```diff
@@ -22,30 +38,31 @@ EOF
...
+    docker run -v /tmp/mc-config:/mc-config minio/mc:edge --config-dir=/mc-config --insecure    mb ai-demo/ai-demo --ignore-existing
 }
 
 function delete_minio_endpoint_route(){
     oc delete route -n minio-operator minio-endpoint
 }
 
+function add_minio_webhook(){
+    # wait until minio-webhook-source Knative Service are ready
+    oc wait --for=condition=Ready ksvc -n ai-demo upload-service
```
Comment (pierDipi) [inline infra/openshift-manifests/lib.sh:51]:
```suggestion
    oc wait --for=condition=Ready ksvc -n ai-demo minio-webhook-source
```

**Comment by aliok** [comment]
It now works, merging this

**Comment by aliok** [comment]
I've actually merged this too early by mistake (didn't contain my latest commit) created #15 for it

**Pull request merged**